api-tests/server/checks_test.go

```diff
@@ -199,21 +201,9 @@ func TestChangeSecurityChecks(t *testing.T) {
 	})
 
 	t.Run("change interval error", func(t *testing.T) {
-		defer restoreSettingsDefaults(t)
```

Comment:
I would add `restoreCheckIntervalDefaults` as cleanup and also check that interval is not changed.

Reply:
Done